tests/crashes/129372.rs

```diff
@@ -1,52 +0,0 @@
-//@ known-bug: #129372
```

Comment:
Would need to properly decide exactly what to do with this test. If just removing it is fine or if there's some variation that needs to be placed in `tests/ui`. Didn't bother to check this for now.